packages/bpk-component-checkbox/src/accessibility-test.js

```diff
@@ -31,4 +29,16 @@ describe('BpkCheckbox accessibility tests', () => {
     const results = await axe(container);
     expect(results).toHaveNoViolations();
   });
+
+  it('should render using label as aria-label if no ariaLabel prop provided', async () => {
+    render(<BpkCheckbox name="checkbox" label="Prefer directs" />);
+    const label = screen.getByLabelText('Prefer directs');
+    expect(label).toBeInTheDocument();
+  });
+
+  it('should render using ariaLabel prop as aria-label if prop provided', async () => {
+    render(<BpkCheckbox name="checkbox" label="Prefer directs" ariaLabel="aria label test string"/>);
+    const ariaLabel = screen.getByLabelText('aria label test string');
+    expect(ariaLabel).toBeInTheDocument();
+  });
```

Comment:
I'd maybe add another test that should render using ariaLabel when both aria-label and label are provided [completely optional to add]